tools/scripts/rebuildDefaultSprites.rb

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env ruby
```

Comment:
super nit: ruby files should be snake case `rebuild_default_sprites.rb` (even though most of the files in this directory don't actually follow that!)